packages/bot/src/assets/icons/icon-info-outline.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16">
```

Comment:
Please use `svgo` to compress this svg

Reply:
@aaron-binary we're not gonna use this svgs, we will get them from icons